src/removeAll.js

```diff
@@ -0,0 +1,16 @@
+export default function removeAll() {
+  const lists = document.querySelectorAll('.todo-list-activities > ul > li');
+  const arrToRemove = [];
+  const previousData = JSON.parse(localStorage.getItem('storedTodos'));
+  for (let i = 0; i < lists.length; i += 1) {
+    if (document.querySelectorAll('.todo-list-activities > ul > li > .checkbox')[i].checked === true) {
+      arrToRemove.push(previousData[i]);
+    }
+  }
```

Comment:
- [ ] It will be nice to implement the `Array.filter()` method to the `removeAll ` method
![Screenshot from 2021-09-23 21-09-05](https://user-images.githubusercontent.com/65476715/134576725-52df5716-6f8c-46c7-907f-26bf1326bc0a.png)